docs/fastdds/logging/consumer.rst

```diff
@@ -22,8 +22,6 @@ StdoutConsumer
 
 |StdoutConsumer-api| outputs log entries to STDOUT stream following the convection specified in
 :ref:`dds_layer_log_logging_spec`.
-It is the default and only log consumer of the logging module if the CMake option |LOG_CONSUMER_DEFAULT| is set to
-``AUTO``, ``STDOUT``, or not set at all.
```

Comment:
Instead of removing this, I would leave it and say that it is the default if the CMake option is set to `STDOUT`